io.github.strozer2002.sparrow-linux.yaml

```diff
@@ -0,0 +1,29 @@
+app-id: io.github.strozer2002.sparrow-linux
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: sparrow
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+modules:
+  - name: sparrow
+    buildsystem: simple
+    build-commands:
+      - cp -R bin/ /app/sparrow
```

Comment:
You need to `mkdir -p /app/sparrow` before copying the files

Reply:
resolved